.pipelines/cni/k8s-e2e/k8s-e2e-step-template.yaml

```diff
@@ -25,6 +25,7 @@ steps:
       # Taint Linux nodes so that windows tests do not run on them
       if ${{ lower(eq(parameters.os, 'windows')) }}
       then
+        kubectl rollout status -n kube-system deployment/konnectivity-agent --timeout=3m
```

Comment:
Is this just for windows, or we need to wait for the deployment in linux as well ?

Reply:
Just for windows due to the linux node being tainted. Stopping scheduling while windows tests are running.